portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,12 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+/**
+* Plays a specified melody
+ */
+function play(melody) {
+    path = "files/audio/";
```

Comment:
Does `path` get defined elsewhere? If not, plz use const to declare. Also, I this isn't the `path`. Maybe move it to the top of the file and name it `AUDIO_DIR`?

Reply:
done!